djangocms_style/models.py

```diff
@@ -1,100 +1,212 @@
-            style += "padding-left: %dpx; " % self.padding_left
+        return self.label or self.tag_type or str(self.pk)
+
+    def get_short_description(self):
+        display = self.tag_type or ''
+        # display the tag correctly
+        if self.label:
+            display = '<{0}> {1}'.format(display, self.label)
+        elif self.class_name:
+            display = '<{0}> .{1}'.format(display, self.class_name)
+        else:
+            display = '<{0}>'.format(display)
+        # display additional information
+        if self.additional_classes:
+            display = '{0} [{1}]'.format(display, self.get_additional_classes)
+        if self.id_name:
+            display = '{0} #{1}'.format(display, self.id_name)
+        return display
+
+    def get_styles(self):
+        styles = []
+        # padding values
+        if self.padding_top:
+            styles.append('padding-top: {0:d}px;'.format(self.padding_top))
         if self.padding_right:
-            style += "padding-right: %dpx; " % self.padding_right
+            styles.append('padding-right: {0:d}px;'.format(self.padding_right))
         if self.padding_bottom:
-            style += "padding-bottom: %dpx; " % self.padding_bottom
-        if self.padding_top:
-            style += "padding-top: %dpx; " % self.padding_top
-        if self.margin_left:
-            style += "margin-left: %dpx; " % self.margin_left
-        if self.margin_right:
-            style += "margin-right: %dpx; " % self.margin_right
+            styles.append('padding-bottom: {0:d}px;'.format(self.padding_bottom))
+        if self.padding_left:
+            styles.append('padding-left: {0:d}px;'.format(self.padding_left))
+        # margin values
         if self.margin_top:
-            style += "margin-top: %dpx; " % self.margin_top
+            styles.append('margin-top: {0:d}px;'.format(self.margin_top))
+        if self.margin_right:
+            styles.append('margin-right: {0:d}px;'.format(self.margin_right))
         if self.margin_bottom:
-            style += "margin-bottom: %dpx; " % self.margin_bottom
-        return style
+            styles.append('margin-bottom: {0:d}px;'.format(self.margin_bottom))
+        if self.margin_left:
+            styles.append('margin-left: {0:d}px;'.format(self.margin_left))
+        return ' '.join(styles)
+
+    def clean(self):
+        # validate for correct class name settings
+        if self.additional_classes:
+            additional_classes = list(
+                html_class.strip() for html_class in self.additional_classes.split(',')
+            )
+            for class_name in additional_classes:
+                class_name = class_name.strip()
+                if not CLASS_NAME_FORMAT.match(class_name):
+                    raise ValidationError(
+                        _('"{name}" is not a proper CSS class name.').format(name=class_name)
+                    )
+            self.additional_classes = ', '.join(set(additional_classes))
```

Comment:
well changing this now will introduce lots of inconsistency, I think this we just have to swallow. Separate individual values by comma.